web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v321/migrate-default.sql

```diff
@@ -1,2 +1,7 @@
 UPDATE Settings SET value='3.2.1' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
+
+ALTER TABLE Settings ADD COLUMN encrypted VARCHAR(1) DEFAULT 'n';
```

Comment:
If there is db structure change, we should probably move that to the migrationMap (to be applied before JPA init), because JPA will also take care of adding the new column and this statement will failed because the column exist already ? I'm not 100% sure... to be tested